proto/desmos/posts/v3/msgs.proto

```diff
@@ -41,6 +41,9 @@ service Msg {
   //
   // Since: Desmos 5.0.0
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+
+   // MovePost allows users to move their own posts to another subspace
```

Comment:
We should probably add a `Since: Desmos 5.1.0` here

Reply:
Maybe Desmos 6.0.0 is better since it would be Desmos v6 feature on roadmap